src/v2/view-builder/views/idp/AuthenticatorIdPView.js

```diff
@@ -0,0 +1,20 @@
+import EnrollIdPBody from './EnrollAuthenticatorIdPBody';
+import ChallengeIdpBody from './ChallengeAuthenticatorIdPBody';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+export default BaseAuthenticatorView.extend({
+  initialize () {
+    BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
+
+    const currentAuthenticator = this.options.appState.get('currentAuthenticator');
+    const authenticatorEnrollments = this.options.appState.get('authenticatorEnrollments')?.value || [];
+    authenticatorEnrollments.filter(authenticator => authenticator.id === currentAuthenticator.id);
+
+    // Set the view based on whether or not we're in a challenge flow
+    this.Body = authenticatorEnrollments.length
```

Comment:
Instead of using length we should use something more concrete to make this decision..

Reply:
Can you provide an example? To me, as long as this value is truthy we're OK proceeding.

Reply:
take a look at DeviceEnrollmentTerminalView.js

Reply:
Hmm. I don't see anything within `DeviceEnrollmentTerminalView` that pertains to this. Can you please link directly to the code or write out an example?

Reply:
Okay.. Got it, i think it should be fine.